python/examples/advanced_agents/assignment-agent/assignment_agent.py

```diff
@@ -0,0 +1,167 @@
+from autogen import AssistantAgent, UserProxyAgent
+from composio_autogen import ComposioToolSet, Action
+from composio import AppType
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+toolset = ComposioToolSet(
+    api_key=os.getenv("COMPOSIO_API_KEY"),
+    entity_id="karthik",
```

Comment:
The `entity_id` is hardcoded as "karthik". Consider making this configurable through environment variables or command-line arguments for better flexibility and reusability.